packages/ember-engines/config/ember-try.js

```diff
@@ -45,6 +45,17 @@ module.exports = async function () {
           }
         }
       },
+      {
+        name: "ember-lts-4.12",
+        npm: {
+          devDependencies: {
+            "ember-source": "~4.8.0",
+            "ember-cli": "~3.28.0",
+            "ember-cli-app-version": "^5.0.0",
+            "@ember/legacy-built-in-components": "~0.4.0",
```

Comment:
starting from this version we really should use latest legacy built in components 0.5.x which has a peer on this ember source

Reply:
good point! Thanks!